client/blocks/comment-detail/comment-detail-edit.jsx

```diff
@@ -85,13 +93,25 @@ export class CommentDetailEdit extends Component {
 				</FormFieldset>
 
 				<FormTextarea
+					disabled={ ! isEditCommentSupported }
 					onChange={ this.setCommentContentValue }
 					value={ commentContent }
 				/>
 
+				{ ! isEditCommentSupported &&
+					<p className="comment-detail__edit-jetpack-update-notice">
+						<Gridicon icon="notice-outline" />
+						{ translate( 'Comment editing requires a newer version of Jetpack.' ) }
+						<a href={ `/plugins/jetpack/${ siteSlug }` }>
+							{ translate( 'Update Now' ) }
+						</a>
+					</p>
+				}
+
 				<div className="comment-detail__edit-buttons">
 					<FormButton
 						compact
+						disabled={ ! isEditCommentSupported }
```

Comment:
Should we also check for this in the Author fields?

Reply:
Damn, good catch!
I was testing on a comment I was sure was from an unregistered user, but it wasn't and I was tricked by the fact that the author fields were already disabled.